app/views/new_admin/regional_holidays/_regional_holidays_table.html.erb

```diff
@@ -10,7 +10,7 @@
   </thead>
   <tbody class="bg-white divide-y dark:divide-gray-700 dark:bg-gray-800 text-sm text-start">
     <% regional_holidays.each do |holiday| %>
-      <tr id=<%= "regional_holiday_#{holiday.id}" %> class="text-gray-700 dark:text-gray-400">
+      <tr id="<%= "regional_holiday_#{holiday.id}" %>" class="text-gray-700 dark:text-gray-400">
```

Comment:
you can use field_id here https://api.rubyonrails.org/classes/ActionView/Helpers/FormBuilder.html#method-i-field_id

Reply:
@adbatista, reading about this method, it seems that it's meant to be used in a ERB form helper, such as `text_field`. Any ideas how it would be assigned to a HTML `<tr>` tag?